### PR TITLE
kishankunal.md

### PR DESCRIPTION
**Pull request by kishankunal**
Suggested Features

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+=======================================================
```
Comment (ajay109458) [inline Readme.md:1]: Rename the file with your name it will help you to refer same in future.

Comment (kishankunal) [reply Readme.md:1]: Done sir

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+1 Functions
+    A. It should contain student information like his name , roll no, registration number and his marks of different        subject of mid term , end term, sesional and of different class test if its considered for creating final weightage.
+    B. Flexible report card models - It should have different TEMPLATES or models to get different styled report card .
+    C. HOD/HOS should be given the right to create/delete exams as well as they can reshedule the exams also.
+    D. Only techer is allowed to enter the marks of student, however HOD/HOs can rectify it if any student wants to          re-check.
+    E. Review/complaint system - student can review their performace and put a complaint if he wants any rectificaton.
+    F. Result Display - Every student would be able to view the marks/grade of students of his own section.
+    G. student should be able to view his marks subject wise. Also he can compare his marks with the top scorer.
```
Comment (ajay109458) [inline Readme.md:13]: Awesome thoughts 👍

**Review by ajay109458 — APPROVED** [review]

**Review by ajay109458 — APPROVED** [review]

File: Readme.md (hunk 2 above)
Comment (kishankunal) [reply Readme.md:13]: thank you sir

**Pull request merged**